include/picongpu/plugins/IsaacPlugin.hpp

```diff
@@ -373,30 +378,16 @@ class IsaacPlugin : public ILightweightPlugin
 
             isaac_size2 framebuffer_size =
             {
-                size_t(width),
-                size_t(height)
+                cupla::IdxType(width),
+                cupla::IdxType(height)
             };
 
             isaac_for_each_params( sources, SourceInitIterator(), cellDescription, movingWindow );
 
-            visualization = new IsaacVisualization
-            <
-                SimDim,
-                SourceList,
-                DataSpace< simDim >,
-                textureDim, float3_X,
-#if( ISAAC_STEREO == 0 )
-                    isaac::DefaultController,
-                    isaac::DefaultCompositor
-#else
-                    isaac::StereoController,
-#   if( ISAAC_STEREO == 1 )
-                        isaac::StereoCompositorSideBySide<isaac::StereoController>
```

Comment:
Do you removed the stereo rendering with this PR?

Reply:
No. The diff looks bigger than it is. I set two variables for findISAAC and removed the "only on GPUs" if.